lua/entities/gmod_wire_expression2/core/serverinfo.lua

```diff
@@ -23,7 +23,7 @@ e2function number isLan()
 end
 
 e2function string gamemode()
-	return gmod.GetGamemode().Name
+	return gmod.GetGamemode().Name or ""
```

Comment:
`Name` could be changed yeah.. This is just to make sure that if Sandbox-derived gamemode is messed up, it will still return an empty string instead, preventing a nil value error (i guess).
It was "just in case" type of thing.